### PR TITLE
Laurie schedule performance charts

### PR DESCRIPTION
**Pull request by lauriemerrell**
## Checklist 
- [ ] I am requesting to merge into the dev branch 
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

## Screenshots

| Before      | After       |
| ----------- | ----------- |
|        |        |

### CODE REVIEW

**Comment by KyleDolezal** [comment]
The only issue I identified is a concern with the performance chart at mobile resolutions.
Screenshot 2023-08-20 at 18 53 19

**Comment by dcjohnson24** [comment]
I set `CI=''` in the `build` command in `package.json`. Not sure whether this is a good idea, but it was suggested [here](https://dev.to/yuridevat/2-ways-to-overcome-deployment-problems-with-react-on-netlify-4l5p#chapter-3).

**Comment by KyleDolezal** [comment]
Regarding the issue I identified [here](https://github.com/chihacknight/ghost-buses-frontend/pull/83#issuecomment-1685430756), I believe we can safely proceed with the code as-is for the following reasons:
- I observed the behavior only on my mac at resolutions near 900x1200. 
- When using more realistic resolutions, those used on Android or iPhones, I cannot see the issue. I tried each of the available phone simulators in Firefox and Chrome.
- I'm also unable to reproduce using resolutions used on most computers. 
- I'm also unable to reproduced at 900x1200 on my Linux machine; it occurs only on my mac.
- I think it unlikely that someone would be using such a resolution in real life.

TLDR: The problem is not reproducible using standard resolutions used by real-life devices.

**Comment by hkier** [comment]
I have not had a chance to really delve into my issues here.  But my big issue has been the inconsistency of the styling between various uses of a component which should be styled the came way in all uses.  This has occurred because of being styled in different files which have become out of sync.  We should be able to merge and then deal with this issue after the merge.
